### DISCUSSION TITLE
Any tips to fast-move in ProjectPanel with keyboards?

### DISCUSSION THREAD

**Discussion by ninthsun91** [zed-industries/zed#14054] [Feature Requests]
I've recently moved to Zed from vim, and one thing I can't find a way to do it right is moving in Project Panel.

I can't find any other way to move between files except j,k — equivalent to arrow keys down&up.

Any tips?

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
